T/tlaplus/build_tarballs.jl

```diff
@@ -0,0 +1,31 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "tlaplus"
+version = v"1.7.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/tlaplus/tlaplus.git", "478d8569e84dfb31b982a500947def5c9c813b97")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    
+]
```

Comment:
I think the resulting JLL package would be empty - you should include whatever files you need to ship as `FileProduct`, `LibraryProduct` or `ExecutableProduct`.

Reply:
Not that much is done in `script` to build anything :sweat_smile:  However, this variable must be `Vector{<:Product}`, so at very least it should be
```julia
products = Product[
]
```